common/lib/api-client/middleware/post.test.js

```diff
@@ -1,6 +1,7 @@
 const FormData = require('form-data')
 
 const postMiddleware = require('./post')
+const { FILE_UPLOADS } = require('../../../../config')
```

Comment:
I find it's best to avoid importing config directly into test files. We've either been using [proxyquire to mock the value](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/app/healthcheck/controllers.test.js#L12) or [injecting config values into the middleware call](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/common/lib/api-client/index.js#L21). Be good to use one of these approaches to stay consistent and to avoid any issues if the config value changes or becomes dependent on an env var.

Reply:
Moved loading config var to main middleware file.